pkg/build/admission/defaults/admission.go

```diff
@@ -1,100 +1,91 @@
 package defaults
 
 import (
-	"io"
-
 	"github.com/golang/glog"
-	"k8s.io/kubernetes/pkg/admission"
 	kapi "k8s.io/kubernetes/pkg/api"
-	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
 
 	buildadmission "github.com/openshift/origin/pkg/build/admission"
 	defaultsapi "github.com/openshift/origin/pkg/build/admission/defaults/api"
 	"github.com/openshift/origin/pkg/build/admission/defaults/api/validation"
 	buildapi "github.com/openshift/origin/pkg/build/api"
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
 )
 
-func init() {
-	admission.RegisterPlugin("BuildDefaults", func(c clientset.Interface, config io.Reader) (admission.Interface, error) {
```

Comment:
it's the only way I can see to make this change and maintain backwards compatibility unless you've got another idea, and @smarterclayton insisted the admission controllers should go away.

i'm going to need you guys to elaborate on this admission chain issue, i don't know what you're referring to.